File: dotnet/src/Pactman.NonprofitCheckPlus/Exceptions/PactmanAuthenticationException.cs
namespace Pactman.NonprofitCheckPlus.Exceptions
{
    /// <summary>HTTP 401. The API key is missing, malformed, revoked or unrecognized.</summary>
    public sealed class PactmanAuthenticationException : PactmanApiException
    {
        /// <summary>Initializes the exception from response metadata.</summary>
        /// <param name="message">Human-readable description of the failure.</param>
        /// <param name="init">The metadata the transport collected from the response.</param>
        public PactmanAuthenticationException(string message, ApiErrorInit init)
            : base(message, init, ErrorCategory.Authentication)
        {
        }
    }
}

File: java/sdk/src/main/java/org/pactman/nonprofitcheckplus/exceptions/PactmanAuthenticationException.java
package org.pactman.nonprofitcheckplus.exceptions;

/** HTTP 401. */
public class PactmanAuthenticationException extends PactmanApiException {

    private static final long serialVersionUID = 1L;

    /**
     * Creates the exception.
     *
     * @param message the message to surface.
     * @param init    the response metadata.
     */
    public PactmanAuthenticationException(String message, ApiErrorInit init) {
        super(message, init, ErrorCategory.AUTHENTICATION);
    }
}
